src/RawTransaction.php

```diff
@@ -1243,6 +1243,17 @@ public static function extract_input_signatures_p2sh($input, $message_hash, $scr
      */
     public static function encode_signature(Signature $signature)
     {
+        $math = EccFactory::getAdapter();
+        $generator = EccFactory::getSecgCurves($math)->generator256k1();
+
+        $n = $generator->getOrder();
+        $against = $n;
+        $against = $math->rightShift($against, 1);
+
+        if ($math->cmp($signature->getS(), $against) < 0 && $math->cmp($signature->getS(), 0) !== 0) {
```

Comment:
```if ($math->cmp($signature->getS(), $against) > 0) {
  $signature = new Signature($signature->getR(), $math->sub($n, $signature->getS()));
}
```

Not sure about S != 0 check, as it's done already in PhpEccs Signer, before the Signature instance is made